stockflux-chart-container/public/app.dev.json

```diff
@@ -0,0 +1,39 @@
+{
+    "runtime": {
+      "arguments": "--enable-aggressive-domstorage-flushing",
+      "version": "9.61.38.40"
+    },
+    "startup_app": {
+      "name": "stockflux-chart-container",
+      "uuid": "stockflux-chart-container",
+      "url": "http://localhost:8051/index.html",
+      "applicationIcon": "{fqdn}/artifacts/stockflux-chart/favicon.ico",
```

Comment:
"applicationIcon": "{fqdn}/artifacts/stockflux-chart-container/favicon.ico",

(still references in this file in two other places)